specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -0,0 +1,5292 @@
+          "enum": [
+            "available",
+            "unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "AccountStatus",
+            "modelAsString": false
+          }
+        },
+        "lastGeoFailoverTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the timestamp of the most recent instance of a failover to the secondary location. Only the most recent timestamp is retained. This element is not returned if there has never been a failover instance. Only available if the accountType is Standard_GRS or Standard_RAGRS."
+        },
+        "secondaryLocation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the location of the geo-replicated secondary for the storage account. Only available if the accountType is Standard_GRS or Standard_RAGRS."
+        },
+        "statusOfSecondary": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status indicating whether the secondary location of the storage account is available or unavailable. Only available if the SKU name is Standard_GRS or Standard_RAGRS.",
+          "enum": [
+            "available",
+            "unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "AccountStatus",
+            "modelAsString": false
+          }
+        },
+        "creationTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the creation date and time of the storage account in UTC."
+        },
+        "customDomain": {
+          "$ref": "#/definitions/CustomDomain",
+          "readOnly": true,
+          "description": "Gets the custom domain the user assigned to this storage account."
+        },
+        "sasPolicy": {
+          "$ref": "#/definitions/SasPolicy",
+          "readOnly": true,
+          "description": "SasPolicy assigned to the storage account."
+        },
+        "keyPolicy": {
+          "$ref": "#/definitions/KeyPolicy",
+          "readOnly": true,
+          "description": "KeyPolicy assigned to the storage account."
+        },
+        "keyCreationTime": {
+          "readOnly": true,
+          "$ref": "#/definitions/KeyCreationTime",
+          "description": "Storage account keys creation time."
+        },
+        "secondaryEndpoints": {
+          "$ref": "#/definitions/Endpoints",
+          "readOnly": true,
+          "description": "Gets the URLs that are used to perform a retrieval of a public blob, queue, or table object from the secondary location of the storage account. Only available if the SKU name is Standard_RAGRS."
+        },
+        "encryption": {
+          "$ref": "#/definitions/Encryption",
+          "readOnly": true,
+          "description": "Gets the encryption settings on the account. If unspecified, the account is unencrypted."
```

Comment:
Update this as well given that all storage accounts are encrypted now

Reply:
make a pass through and ensure all the encryption related descriptions are accurate.  These description errors are littered throughout

Reply:
Updated the description for the encryption properties in the swagger yo reflect current state.